utilities/check_url.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Ensure a separate script is called in the event of an error
+#
+# Include in your builds via
+# \curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/utilties/check_url.sh > ${HOME}/bin/check_url && chmod u+x ${HOME}/bin/check_url
+#
+# then use the script in your tests like
+# check_url -t 6 -w 10 -o "--no-check-certificate --output-document=/dev/null" "url_to_check"
+OPTIONS="--no-check-certificate --output-document=/dev/null"
```

Comment:
`--no-check-certificate` should be _opt-in_ and only for really good reasons...

Some reading material to convince you:
- http://mislav.uniqpath.com/2013/07/ruby-openssl/
- https://blog.cloudflare.com/introducing-strict-ssl-protecting-against-a-man-in-the-middle-attack-on-origin-traffic/
- https://news.ycombinator.com/item?id=8555026